packages/private-build-infra/src/addon-build-config-for-data-package.js

```diff
@@ -45,35 +45,28 @@ function addonBuildConfigForDataPackage(PackageName) {
       }
     },
 
-    _suppressUneededRollupWarnings(message, next) {
+    _handleRollupWarning(message) {
       if (message.code === 'CIRCULAR_DEPENDENCY') {
         return;
-      } else if (message.code === 'NON_EXISTENT_EXPORT') {
+      } else if (message.code === 'NON_EXISTENT_EXPORT' && message.message.includes('ts-interfaces')) {
         // ignore ts-interface imports
-        if (message.message.indexOf(`/ts-interfaces/`) !== -1) {
-          return;
-        }
-      } else if (message.code === 'UNRESOLVED_IMPORT') {
-        if (!this.isDevelopingAddon()) {
```

Comment:
we shouldn't remove this check as rollup, babel, rollup plugins, broccoli, and broccoli-rollup may all be a different version in a consuming app than what we test and ship with. It's better to print an unnecessary warning than to break the build for those folks.